src/common/opt.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _BLUE_CHIHUAHUA_NODE_OPTIONS_COMMON
+#define _BLUE_CHIHUAHUA_NODE_OPTIONS_COMMON
+
+#define ARG_PORT "port"
+#define ARG_PORT_SHORT 'P'
+#define ARG_HOST "host"
+#define ARG_HOST_SHORT 'H'
+#define ARG_CONFIG "config"
+#define ARG_CONFIG_SHORT 'c'
```

Comment:
No, but I have seen the usage of capital letters in many projects, even specifically for host/port. -h/--help should be help, because it is the first arg that people try with CLI, and when it does not return what you expected (help) - it's a frustration. From there, if you need the letter H for something else, you can use the capital one, and it's okay. So it happens with "host" a lot, so since host is capital H, I figured port should be capital as well

Reply:
Yea, this is merely convention, but as Mark said, this is common in C code.